Xamarin.PropertyEditing.Mac/Controls/Custom/NumericSpinEditor.cs

```diff
@@ -249,24 +244,24 @@ void ThemeManager_ThemeChanged (object sender, EventArgs e)
 
 		protected void UpdateTheme ()
 		{
-			this.Appearance = PropertyEditorPanel.ThemeManager.CurrentAppearance;
+			Appearance = PropertyEditorPanel.ThemeManager.CurrentAppearance;
 		}
 
 		virtual protected void OnEditingEnded (object sender, EventArgs e)
 		{
-			if (!editing) {
-				editing = true;
+			if (this.editing) {
```

Comment:
Looks like the ! got dropped in adding `this`

Reply:
Fixed.